packages/nodes-base/nodes/Aws/Transcribe/AwsTranscribe.node.ts

```diff
@@ -438,9 +439,8 @@ export class AwsTranscribe implements INodeType {
 							Object.assign(body.Settings, { ChannelIdentification: options.channelIdentification });
 						}
 
-						if (options.MaxAlternatives) {
+						if (options.maxAlternatives) {
 							Object.assign(body.Settings, {
-								ShowAlternatives: options.maxAlternatives,
```

Comment:
I think this attribute is required with a boolean value, as mentioned in the [docs](https://docs.aws.amazon.com/transcribe/latest/dg/API_Settings.html#transcribe-Type-Settings-MaxAlternatives), also the same for `MaxSpeakerLabels` , also they mentioned that you can't set `ChannelIdentification` with `ShowSpeakerLabels` but that it not handled in the node.
![image](https://user-images.githubusercontent.com/36171143/120658893-22984b80-c486-11eb-90e4-d288469868bf.png)

Reply:
Also you can do the same attributes name changes in line 448 switch it with 'maxSpeakerLabels'.